app/pom.xml

```diff
@@ -115,7 +115,8 @@
     <json-schema-validator.version>2.2.8</json-schema-validator.version>
     <junit.version>4.12</junit.version>
     <powermock.version>2.0.0</powermock.version>
-    <testcontainers.version>1.10.6</testcontainers.version>
+    <testcontainers.version>1.10.7</testcontainers.version>
+    <citrus.version>2.7.8</citrus.version>
```

Comment:
Do we have troubles using the last bits in central?

Reply:
Is your question related to citrus version? We can not use the latest 2.8.x versions here because this is using Spring Boot 2.x I think in Syndesis we are still bound to Spring 1.5.x so we need to use citrus 2.7.8